index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
             </li>             
      </ul>               
     </section>
+    <section class="works">
+        <div class="title">
+           <h2>My Recent Works</h2>
+           <div class="line"></div>
+        </div>
+        <div class="sample">
+           <img src="./assets/images/works-section.png" alt="works">
+           <h3>Multi-Post Stories</h3>
+           <p class="works-para">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="tags">
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">HTML</li>
+            <li class="tag-item">Bootstrap</li>
+            <li class="tag-item">Ruby</li>
+           </ul>
+           <button type="button">See Project</button>
+        </div>
+        <div class="works-grid">
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+           </div> 
```

Comment:
- [ ] According to the project requirements, kindly note that you are required to use flexbox to the individual containers of the projects of this section. Currently, I can see that you are not making use of the display: flex; for this container wrapper. Kindly include it.

**Screenshot from the project requirements:**

![image](https://user-images.githubusercontent.com/86318284/185755831-9e5a3864-d932-4338-9cd9-c38b595d1d7e.png)